website/docs/docs/core/connect-data-platform/snowflake-setup.md

```diff
@@ -98,7 +98,8 @@ Along with adding the `authenticator` parameter, be sure to run `alter account s
 
 ### Key Pair Authentication
 
-To use key pair authentication, omit a `password` and instead provide a `private_key_path` and, optionally, a `private_key_passphrase` in your target. **Note:** Versions of dbt before 0.16.0 required that private keys were encrypted and a `private_key_passphrase` was provided. This behavior was changed in dbt v0.16.0.
+To use key pair authentication, skip the `password` and provide a `private_key_path`. If needed, you can also add a `private_key_passphrase`. 
+**Note**: In dbt versions before 0.16.0, private keys needed encryption and a `private_key_passphrase`. From dbt version 0.16.0 onwards, unencrypted private keys are accepted, so add a passphrase only if necessary.
```

Comment:
Removing the "prior to .16.0" language since this is long since deprecated

```suggestion
**Note**: Unencrypted private keys are accepted, so add a passphrase only if necessary.
```